.github/workflows/gradle.yml

```diff
@@ -49,8 +49,6 @@ jobs:
           [ -f ./setup.sh ] && ./setup.sh || true
       - name: Build with Gradle
         run: |
-          # Awful hack for kapt and JDK 16. See https://youtrack.jetbrains.com/issue/KT-45545
-          if [ ${{ matrix.java }} == 16 ]; then export GRADLE_OPTS="-Dorg.gradle.jvmargs=--illegal-access=permit"; fi
```

Comment:
are we positive we don't need this anymore?

Reply:
According to https://youtrack.jetbrains.com/issue/KT-45545 it is marked as fixed as of kotlin 1.6.0.and Gradle 7. We are using newer/latest versions of both.